docs/rbac.rst

```diff
@@ -0,0 +1,269 @@
+Role Based Access Control
+=========================
+
+.. important::
+
+    This feature is a Tech Preview and can be subject of change.
+
+.. important::
+
+    Using user ``admin`` will bypass all RBAC checks.
+
+
+* `Permission` : Permission to do **one** action on one `viewset` as a `view` or a `create`.
+
+  * `Model Permission` : a permission for a model type, it means you can do this action on any object of the model
```

Comment:
Are these docs meant to be for the user or for the plugin writer? If they're for a user, I would avoid using terms like "model" and "viewset".